### DISCUSSION TITLE
New store / product (not video card or Xbox)

### DISCUSSION THREAD

**Discussion by unekgal** [jef/streetmerchant#1712] [Help]
I'm looking to tweak this script to track a current hard to find item (not a video card or Xbox). 

I have edited the required files below, but I'm getting errors. Are there other files that need edited since I'm not searching for a card or Xbox? Example item: https://www.midwayusa.com/product/1022339657?pid=858296

Any assistance would be greatly alleviated, thanks!

1. /streetmerchant/.env
2. /streetmerchant/src/store/model/store.ts
3. /streetmerchant/src/store/model/newegg.ts

**Comment by imsnot** [comment]
https://jef.codes/streetmerchant/help/general/
